great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -150,34 +156,48 @@ def get_line_chart(
         Returns:
             An altair line chart
         """
-        metric_title: str = metric_name.replace("_", " ").title()
-        domain_title: str = domain_name.title()
-
-        title: Union[str, alt.TitleParams] = f"{metric_title} per {domain_title}"
-        if subtitle:
-            title = alt.TitleParams(title, subtitle=[subtitle])
+        metric_component: MetricPlotComponent = MetricPlotComponent(
+            metric_name, metric_type
+        )
+        domain_component: DomainPlotComponent = DomainPlotComponent(
+            domain_name, domain_type
+        )
+        batch_id_component: BatchPlotComponent = BatchPlotComponent(
+            "batch_id", AltairDataTypes.NOMINAL.value
+        )
+        return DataAssistantResult._get_line_chart(
+            df=df,
+            metric_component=metric_component,
+            domain_component=domain_component,
+            batch_id_component=batch_id_component,
+            subtitle=subtitle,
```

Comment:
Can `subtitle` become an optional property of `DomainPlotComponent`? Right now, a subtitle is needed for all domain types except `MetricDomainTypes.TABLE`.